charts/gsp-cluster/templates/02-gsp-system/service-operator/deployment.yaml

```diff
@@ -0,0 +1,40 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}-service-operator
+  labels:
+    app.kubernetes.io/name: "service-operator"
+    helm.sh/chart: {{ .Chart.Name }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    control-plane: service-operator
+    controller-tools.k8s.io: "1.0"
+spec:
+  selector:
+    matchLabels:
+      control-plane: service-manager
```

Comment:
This should include more templating; we don't want to run the risk of selecting additional pods.